src/USER-CONP/fix_charge_update.cpp

```diff
@@ -274,6 +289,26 @@ void FixChargeUpdate::invert(std::vector<std::vector<double>> capacitance) {
 
 /* ---------------------------------------------------------------------- */
 
+void FixChargeUpdate::symmetrize() {
+  // S matrix to enforce charge neutrality constraint
+  std::vector<double>AinvE(ngroup, 0.);
+  double EAinvE = 0.0;
+  for (int i = 0; i < ngroup; i++) {
+    for (double e : elastance[i]) {
```

Comment:
Diese Notation kenne ich gar nicht... was macht die?

Reply:
Das ist eine Range-Loop. Das ist equivalent zu:
`for (int j = 0; j < elastance.size(); j++) {
    double e = elastance[j]
    ...
}`
Funktioniert aber nur mit Objekten bei denen die Größe klar definiert ist als z.B. std::vector

Reply:
Du machst Sachen :). Naja, so lange das C++11 konform ist, ist alles cool!

Reply:
Hab nachgeguckt es ist zum Glück in c++11.
So abgefahrens ist es nicht. Quasi das gleich wie `for x in list:` in Python.